<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlgebraPage</class>

 <widget class="QWidget" name="AlgebraPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout">

   <!-- Title -->
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Algebra</string>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>

   <!-- Subtitle -->
   <item>
    <widget class="QLabel" name="labelSubtitle">
     <property name="text">
      <string>Solve quadratic equations of the form ax² + bx + c = 0</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>

   <!-- Inputs -->
   <item>
    <widget class="QGroupBox" name="groupBoxInputs">
     <property name="title">
      <string>Inputs</string>
     </property>

     <layout class="QFormLayout" name="formLayoutInputs">
      <item row="0" column="0">
       <widget class="QLabel" name="labelA">
        <property name="text">
         <string>a</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditA"/>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="labelB">
        <property name="text">
         <string>b</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditB"/>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="labelC">
        <property name="text">
         <string>c</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditC"/>
      </item>

      <!-- Solve Button -->
      <item row="3" column="0" colspan="2">
       <widget class="QPushButton" name="buttonSolve">
        <property name="text">
         <string>Solve</string>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <!-- Results -->
   <item>
    <widget class="QGroupBox" name="groupBoxResults">
     <property name="title">
      <string>Results</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
     </property>

     <layout class="QGridLayout" name="gridLayoutResults">
      <item row="0" column="0">
       <widget class="QLabel" name="labelX1">
        <property name="text">
         <string>x₁</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelX1Value">
        <property name="text">
         <string>—</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="labelX2">
        <property name="text">
         <string>x₂</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelX2Value">
        <property name="text">
         <string>—</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

  </layout>
 </widget>

 <resources/>
 <connections/>
</ui>
